### PR TITLE
gha: Implement cron and manual jobs

### PR DESCRIPTION
**Pull request by mauriciovasquezbernal**
- Implement a daily cron job to keep documents updated.
- Use workflow_dispatch to manually dispatch a job.

### CODE REVIEW

**Comment by alban** [comment]
There is still the following check a few lines below, so it would not publish automatically, right?
```
if: github.event_name == 'push' ||
```

**Comment by mauriciovasquezbernal** [comment]
You're right. Updated it to include both events on such conditional.

File: .github/workflows/azure-static-web-apps-polite-mushroom-00d947803.yml [hunk 1]
```diff
@@ -8,10 +8,16 @@ on:
...
+    # Run at 00:01 daily
+    - cron: '01 0 * * *'
+
+  workflow_dispatch:
+    # For manual dispatch
 
 jobs:
   build_and_deploy_job:
-    if: github.event_name == 'push' || (github.event_name == 'pull_request' && github.event.action != 'closed')
+    if: github.event_name == 'schedule' || github.event_name == 'workflow_dispatch' || github.event_name == 'push' || (github.event_name == 'pull_request' && github.event.action != 'closed')
```
[the remark this answers is not in the document]
Comment (mauriciovasquezbernal) [reply .github/workflows/azure-static-web-apps-polite-mushroom-00d947803.yml:20]: It publishes to the main website.

[the remark this answers is not in the document]
Comment (mauriciovasquezbernal) [reply .github/workflows/azure-static-web-apps-polite-mushroom-00d947803.yml:20]: TBH I'm not totally sure how it works. This setup appears to be taken from the official documentation -> https://learn.microsoft.com/en-us/azure/static-web-apps/build-configuration?tabs=github-actions#build-configuration. 

I suppose the internal action code checks if the event is a PR or a commit to main, and based on that it decides where to deploy?

**Comment by mauriciovasquezbernal** [comment]
It was implemented in a different way in https://github.com/inspektor-gadget/website/pull/15.

**Pull request closed without merging**